test/adapter/polymorphic_test.rb

```diff
@@ -0,0 +1,72 @@
+require 'test_helper'
+
+module ActiveModel
+  class Serializer
+    module Adapter
```

Comment:
IMHO we should split this into different files under the adapters namespace for that sake of organization